tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -2225,6 +2251,36 @@ def _forward_step(self,
+        if getattr(scheduled_requests.context_requests[0], 'multimodal_lengths', None) is None:
+            multimodal_chunks = None
+        else:
+            multimodal_chunks = [
+                sum(request.multimodal_lengths)
+                for request in scheduled_requests.context_requests
+                if request.multimodal_lengths is not None
+            ]
+        # For mm_encoder_only mode, we only run the vision encoder part
+        # The model should be a vision encoder (e.g., Qwen2VisionModelBase)
+        mm_embeddings = self.model.forward(multimodal_params)
+        assert len(mm_embeddings) == 1, "mm_embeddings should be a 1-element list, mix modality (video+image) is not supported"
```

Ruff (0.12.2)

Line too long (127 > 120)

(E501)